apis/src/websockets/api/chat/handler.rs

```diff
@@ -70,6 +70,10 @@ impl ChatHandler {
                     message: ServerMessage::Chat(vec![self.message.to_owned()]),
                 })
             }
+            ChatDestination::Global => messages.push(InternalServerMessage {
```

Comment:
This needs a check that the user who's sending the global message is actually allowed to do so, so please check that they are an admin, here.